README.md

```diff
@@ -18,10 +18,121 @@ Like moustache / handlebars but with FEEL.
 
 
 ## Usage 
-...
+
+Feelers is a string templating tool, and will return string text or error.
+
+```js
+import { evaluate } from 'feelers'`
+```
+A simple string will always be returned as-is.
+
+```js
+evaluate("My simple string");
+// "My simple string"
+```
+
+If your string is prefixed with an =, it will be evaluated as a single FEEL expression.
+
+```js
```

Comment:
I find this aspect interesting and would expect, as a template engine, feelers only operates on strings, never on expressions.

This ties to the spirit of "do one thing and do it right": I'd rather not have it in feelers, but outside. This otherwise ties feelers to core features some random engine or tooling decided upon.

My cents. Maybe there is room to improve this.

----

Adding this PR (and the insights I gained reviewing it) is a great example for readme driven development, and how explaining before building can help us to shape ideas (and build simple, focused, and easy to understand things.

Reply:
@Skaiir Did you consider my comment, i.e. does it make sense?